doc/3/controllers/collection/create/index.md

```diff
@@ -21,13 +21,13 @@ This method will only update the mapping if the collection already exists.
 ## Arguments
 
 ```java
-public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+public void create(
```

Comment:
This makes this method sync, returning before the query is sent to Kuzzle. If there is a problem, the requesting user cannot know it.

Instead, consider returning a `CompletableFuture<Void>`